man/logind.conf.xml

```diff
@@ -212,6 +212,14 @@
         <xi:include href="version-info.xml" xpointer="v256"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>HandleSecureAttentionKey=</varname></term>
+
+        <listitem><para>Specifies if the Secure Attention Sequence (Ctrl+Alt+Shift+Esc) should be handled or not</para>
```

Comment:
drop the "or not" please, and suffix this with a full stop. 

And say what the default is.